django/db/models/expressions.py

```diff
@@ -1110,6 +1126,13 @@ def as_sql(self, compiler, connection):
         return "*", []
 
 
+class DB_DEFAULT(Expression):
```

Comment:
I would treat this as a constant
```suggestion
class DatabaseDefault(Expression):
```